packages/blade/src/components/Typography/BaseText/BaseText.tsx

```diff
@@ -0,0 +1,45 @@
+import type { ReactElement } from 'react';
+import getIn from '../../../utils/getIn';
+import { useTheme } from '../../BladeProvider';
+import isEmpty from '../../../utils/isEmpty';
+import StyledBaseText from './StyledBaseText';
+import type { BaseTextProps } from './BaseText.d';
+
+const BaseText = ({
+  color,
+  fontFamily,
+  fontSize,
+  fontWeight,
+  fontStyle,
+  textDecorationLine,
+  lineHeight,
+  as,
+  children,
+  textAlign,
+}: BaseTextProps): ReactElement => {
+  const { theme } = useTheme();
+  const textColor = getIn(theme.colors, color);
+
+  if (isEmpty(textColor)) {
+    throw new Error(`[Blade:BaseText]: Invalid value ${color} for color prop passed`);
```

Comment:
Are you getting

```
Invalid type "BaseTextProps" of template literal expression.
```
here?

Reply:
when?

Reply:
Fine now with the getInvalidColorPropValueError function